### PR TITLE
Initial dutch translation

### PR DESCRIPTION
**Pull request by supakeen**
Provide Dutch translations as mentioned in #560 

I've updated the other README.* and CONTRIBUTING.* to refer to this new translation. Are these supposed to be in alphabetical order? I've currently added Dutch to the bottom of each.

Thanks for your work.

### CODE REVIEW

**Comment by wopian** [comment]
Thank you!

>Are these supposed to be in alphabetical order?

Yes

**Comment by supakeen** [comment]
Updated the language listings to be in alphabetical order :)

File: README.nl.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+### Contract en Goedkeuring 
+*Afgeleid van [CONTRACT_AND_APPROVALS.agc]*
+
+Dit AGC programma zal worden benoemd als Colossus 2A.
+
+Dit programma is bedoeld voor gebruik in de CM zoals vastgesteld in rapport `R-577`. Dit programma is voorbereid onder DSR project `55-23870`, gesponsord door het Manned Spacecraft Center van de National Aeronautics and Space Administration via contract `NAS 9-4065` met het Instrumentation Laboratory, Massachusetts Instute of Technology, Cambridge, Mass.
+
+Ingediend door        | Rol  | Datum
+:-------------------- | :--- | :---
+Margaret H. Hamilton  | Colossus Programming Leader<br>Apollo Guidance and Navigation | 28 Mar 69
```
Comment (wopian) [inline README.nl.md]: `28 maart 69` or `28 mrt 69`?

File: README.nl.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+Goedgekeurd door   | Rol  | Datum
+:----------------- | :--- | :---
+Daniel J. Lickly   | Director, Mission Program Development<br>Apollo Guidance and Navigation Program | 28 Mar 69
```
Comment (wopian) [inline README.nl.md]: Possibly translate `Director`, `[Colossus] Project Manager` and `Deputy Director`

File: README.nl.md (hunk 1 above)
Comment (supakeen) [reply README.nl.md]: You're right, I had looked at the Spanish translation for reference where the dates are not translated. I've added a commit to translate the dates.

File: README.nl.md (hunk 2 above)
Comment (supakeen) [reply README.nl.md]: I'll have to ask around about this one, I'm uncertain if the Dutch terms for these positions carry the same meaning I'll update this after!

Comment (supakeen) [reply README.nl.md]: I've translated those that have direct Dutch translations and left the one (Programming Leader) which has no direct translation. If you want me to be a bit more liberal and translate that one as well let me know!

**Comment by wopian** [comment]
Ready to merge if everything is done 👍

**Comment by supakeen** [comment]
All good, ready to merge at your leisure. Thanks for the review and comments.

**Pull request merged**